src/modules/ProductLicense.rb

```diff
@@ -377,14 +379,15 @@ def GetLicenseDialog(languages, license_language, licenses, id, spare_space)
               ReplacePoint(
                 Id(:printing_hint),
                 Label(
-                  # TRANSLATORS: addition license information
-                  # %1 is replaced with the filename
-                  Builtins.sformat(
-                    _(
-                      "If you want to print this EULA, you can find it\non the first media in the file %1"
-                    ),
-                    @license_file_print
-                  )
+                  @license_on_installed_system ?
+                      # TRANSLATORS: addition license information
+                      # %s is replaced with the directory name
+                      _("This EULA can be found in the directory %s") % @license_file_print
```

Comment:
NP: As currently discussed on yast-devel@, I'd put the directory name on a separate line to make sure it fits the dialog in text mode.